site.yml

```diff
@@ -60,3 +60,30 @@
   - { role: swarm, run_as: worker }
   - { role: contiv_network, run_as: worker }
   - { role: contiv_storage, run_as: worker }
+
+# net-master hosts correspond to cluster machines that run the master
+# logic of the contiv net services
+- hosts: net-master
+  sudo: true
+  environment: env
+  roles:
+  - { role: base }
```

Comment:
you missed the point. To put it the other way around, if you use `net-master` host-group for provisioning a vagrant environment it would run `base` role twice, right? 

The `devtest` role is used for setting up basic dev environment on bare-metals (by `contiv/lab`) and for vagrant box images (by `contiv/build`). So assuming, the new host-groups are built on top of such machines then `base` should not be included again in any of the host-groups that provision things on top of it otherwise we are unnecessary running same ansible twice. Again this is not an optimization but more of how things are setup right now.